.github/workflows/markdownLint.yaml

```diff
@@ -0,0 +1,34 @@
+name: Markdown Lint
+
+on:
+  push:
```

Comment:
Will this run the linter twice? Once when a PR is opened and once when it's merged into main?  If so, we should probably ignore pushes to `main`